.github/workflows/testing.yml

```diff
@@ -639,6 +639,12 @@ jobs:
             registry.taichigraphics.com/taichidev-androidsdk18.04:v0.0.6 \
             /home/dev/taichi/.github/workflows/scripts/aot-demo.sh smoke-test-unity-demo
 
+      - name: Save C-API shared object (Android)
+        uses: actions/upload-artifact@v3
+        with:
+          path: build/libtaichi_c_api.so
```

Comment:
@feisuzhu OOC is there a fixed URL that other CI can download from or do they need to find out the link manually? ;)

Reply:
No, there's no fixed url :(